packages/edit-site/src/components/sidebar/global-styles/screen-header.js

```diff
@@ -0,0 +1,49 @@
+/**
+ * WordPress dependencies
+ */
+import {
+	__experimentalHStack as HStack,
+	__experimentalVStack as VStack,
+	__experimentalSpacer as Spacer,
+	__experimentalHeading as Heading,
+	__experimentalView as View,
+} from '@wordpress/components';
+import { isRTL } from '@wordpress/i18n';
+import { chevronRight, chevronLeft, Icon } from '@wordpress/icons';
+
+/**
+ * Internal dependencies
+ */
+import NavigationButton from './navigation-button';
+
+function ScreenHeader( { back, title, description } ) {
+	return (
+		<VStack spacing={ 2 }>
+			<HStack spacing={ 2 }>
+				<View>
+					<NavigationButton
+						path={ back }
+						icon={
+							<Icon
+								icon={ isRTL() ? chevronRight : chevronLeft }
+								variant="muted"
+							/>
+						}
+						size="small"
+						isBack
+					/>
+				</View>
+				<Spacer>
+					<Heading level={ 5 }>{ title }</Heading>
```

Comment:
In terms of content hierarchy, should we expose the heading's level as a prop, so that it can be changed depending on the level of nesting of the screen?

Reply:
Same would apply to the `Subtitle` component in this PR

Reply:
> In terms of content hierarchy, should we expose the heading's level as a prop, so that it can be changed depending on the level of nesting of the screen?

No, I think that would be the wrong approach, I'd rather we normalize these across all global styles sidebar screens. All header should be the same and all subheaders should be the same. Let's see how it evolves and if we really need more levels.

Reply:
Makes sense